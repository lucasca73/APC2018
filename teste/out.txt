Novo salario: 460.00
reajuste ganho: 60.00
Em percentual: 15 %
Novo salario: 880.01
reajuste ganho: 80.00
Em percentual: 10 %
Novo salario: 2140.00
reajuste ganho: 140.00
Em percentual: 7 %
