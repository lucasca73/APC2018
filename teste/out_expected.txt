Novo salario: 460.00
Reajuste ganho: 60.00
Em percentual: 15 %
Novo salario: 880.01
Reajuste ganho: 80.00
Em percentual: 10 %
Novo salario: 2140.00
Reajuste ganho: 140.00
Em percentual: 7 %
